src/tiled/imagecolorpickerwidget.cpp

```diff
@@ -0,0 +1,106 @@
+#include "imagecolorpickerwidget.h"
+#include "ui_imagecolorpickerwidget.h"
+
+#include <QString>
+#include <QFileInfo>
+#include <QDesktopWidget>
+
+using namespace Tiled;
+using namespace Tiled::Internal;
+
+ImageColorPickerWidget::ImageColorPickerWidget(QWidget *parent) :
+    QWidget(parent),
+    mUi(new Ui::imageColorPickerWidget)
+{
+    mUi->setupUi(this);
+    connect(mUi->imageArea, SIGNAL(mouseMoved(QMouseEvent*)), SLOT(onMouseMove(QMouseEvent*)));
+    connect(mUi->imageArea, SIGNAL(mousePressed(QMouseEvent*)), SLOT(onMousePress(QMouseEvent*)));
+    connect(mUi->imageArea, SIGNAL(mouseReleased(QMouseEvent*)), SLOT(onMouseRelease(QMouseEvent*)));
+    mPreviewIcon = QPixmap(128, 32);
+}
+
+ImageColorPickerWidget::~ImageColorPickerWidget()
+{
+    delete mUi;
+}
+
+bool ImageColorPickerWidget::selectColor(const QString &image)
+{
+    QPixmap pix(image);
+    if (!pix.isNull())
+    {
```

Comment:
Except when defining functions, I always place the `{` on the same line. Please adjust the code accordingly.